11ty/_includes/layouts/base.njk

```diff
@@ -3,17 +3,20 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
+    <link rel="preconnect" href="https://use.typekit.net/" crossorigin>
+    <link rel="dns-prefetch" href="https://use.typekit.net/">
+    <link rel="preconnect" href="https://p.typekit.net/" crossorigin>
+    <link rel="dns-prefetch" href="https://p.typekit.net/">
     {# Use title with path, or append a space to the page title to avoid collpasing with the meta title #}
     {% set pageTitle = titleWithPath or title + ' ' or '' %}
     <title>{{ pageTitle + metadata.title }}</title>
     <meta
       name="description"
       content="{{ renderData.description or description or metadata.description }}"
     >
-    <link rel="stylesheet" href="https://use.typekit.net/qlo3dpu.css" rel="preload"/>
```

Comment:
This is currently not the correct syntax for preloading a stylesheet, so I removed the `rel` attribute here. If we were to preload this stylesheet, it would be in a separate `<link>` tag that looks like this:

```html
<link rel="preload" as="style" href="https://use.typekit.net/qlo3dpu.css" crossorigin>
```

Even so, the CSS is so far up in the head that preloading it will have no practical effect. The main bottleneck in font loading is waiting for the browser to connect to p.typekit.net after it discovers the `@import` in the CSS loaded from use.typekit.net.

Reply:
I agree. The CSS payload size seems to be small enough to hold off on `media="print"` trickery for now.